Update oauth_for_integrations.md (#26448)
DataDog/documentation: content/en/developers/integrations/oauth_for_integrations.md

Until an OAuth client is published, you can only authorize the client from the account it was created in (the partner's sandbox account). This error occurs if you try to authorize the client outside of that account before the client is published.

> If you've already published your OAuth client, make sure to use the client ID and client secret that you received upon submission. The client secret is displayed only once. If you've lost your client secret, contact [ecosystems@datadog.com][11] for assistance.

Reviewer: ```suggestion
If you've already published your OAuth client, remember to use the client ID and the client secret you were given at submission. The client secret was displayed only once, so if you've lost it, contact [ecosystems@datadog.com][11] for assistance.
```

- Confirm you are using the correct redirect URI. For example, if your client is published, the redirect URI should match the one configured for production, and not the URI you used for testing.
- Ensure you are using the correct client. Use your testing client until the integration is published to your sandbox account.

### Applications with subdomains

> We do not support multi-tenanted applications where customers have individual subdomains to authorize against. We only support authorizing against a single domain. 

Reviewer: ```suggestion
Datadog does not support multi-tenanted applications where customers authorize using individual subdomains; instead, authorization is supported only through a single domain.
```
